documentation/source/LRAssignment.md

```diff
@@ -0,0 +1,116 @@
+# Assigning Learning Rates in SG
+The `initial_lr` training hyperparameter allows you to specify different learning rates for different layers or groups of parameters in your neural network. This can be particularly useful for fine-tuning pre-trained models or when different parts of your model require different learning rate settings for optimal training.
+
+## Using initial_lr as a Scalar:
+When initial_lr is a single floating-point number, it sets a uniform learning rate for all model parameters. For example, initial_lr = 0.01:
```

Comment:
When referring to a variable name from the text, it's cool to use ` quotes to highlight the variable itself: `initial_lr`, `initial_lr = 0.01`